### PR TITLE
MIMP-17 Fix funnel plot bug

### PR DESCRIPTION
**Pull request by BalancedmonkeY**
Fixed a bug in plotting the simulated trials on the extended funnel plot (details in Jira MIMP-17 and #18 ).
Whilst working in file `SampleSizeFunctions.R`, improved function documentation.

Further improvements that were linked to the fix (saving the simulation data to text files) has been documented as a task Jira MIMP-20.

### CODE REVIEW

**Review by JanionNevill — CHANGES_REQUESTED** [review]
Minor changes to the Roxygen comments.

I tried following the reproduction steps in the Jira issue, but I'm getting a different error before even hitting the end of the steps:
```
Error in metapow: unused arguments (chains = chains(), iter = iter(), warmup = burn(), prior = prior())
```

File: R/SampleSizeFunctions.R [hunk 1]
```diff
@@ -1,44 +1,32 @@
...
+#' @param measure type of outcome that is being used for the meta-analysis (or/rr/rd/md)
+#' @return A list containing the key data for the newly simulated trial
+#'  - 'new.events.t': For binary outcome, number of events in the treatment arm of the new trial
+#'  - 'new.events.c': For binary outcome, number of events in the control arm of the new trial
+#'  - 'new.effect': For binary outcome, new effect result for new trial (e.g. OR)
+#'  - 'new.mean.t': For continuous outcome, new mean in treatment arm of new trial
+#'  - 'new.stdev.t': For continuous outcome, new standard deviation in treatment arm of new trial
+#'  - 'new.mean.c': For continuous outcome, new mean in control arm of new trial
+#'  - 'new.stdev.c': For continuous outcome, new standard deviation in control arm of new trial
+metasim <- function(es, tausq, var, model, n, data, measure) {
```
[the remark this answers is not in the document]
Comment (BalancedmonkeY) [reply R/SampleSizeFunctions.R:23]: It should do? I've clarified that the contents are different depending on `measure`.

But it should be returned by 
`new <- list(new.events.t = new.events.t, new.events.c = new.events.c, new.effect = new.effect)` or `new <- list(new.mean.t = new.mean.t, new.stdev.t = new.stdev.t, new.mean.c = new.mean.c, new.stdev.c = new.stdev.c)`, then `return(new)`

[the remark this answers is not in the document]
Comment (JanionNevill) [reply R/SampleSizeFunctions.R:25]: `CutOffSettings()` at the bottom still needs Roxygen comments fixing. Sorrythaaaanks

**Review by JanionNevill — APPROVED** [review]
Just one other Roxygen fix

**Pull request merged**